.github/workflows/plugin-delivery.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     runs-on: [self-hosted, common]
```

Comment:
Shouldn't we also change it to ubuntu

Reply:
Soon I guess
But currently delivery is possible only from internal network